page-content/Index.html

```diff
@@ -30,8 +30,8 @@ <h2>NER Project Management</h2>
           aria-selected="false">Gantt</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" id="wp-lookup-tab" data-toggle="tab" href="#wp-lookup" role="tab" aria-controls="wp-lookup"
-          aria-selected="false">WP Lookup</a>
+        <a class="nav-link" id="lookups-tab" data-toggle="tab" href="#lookups" role="tab" aria-controls="lookups"
+          aria-selected="false">Projects/WP Lookup</a>

```

Comment:
Do you think the page should just be called something else?

Reply:
looks good to me, but I would make the tab title just be "Lookup"

Reply:
Resolved by commit 799059001e5d1e3e9d2042af8855a80374c2623d